frontend/src/features/projects/assess/steps/ReviewApproveStep.tsx

```diff
@@ -92,14 +95,20 @@ const ReviewApproveStep = ({ formikRef }: IStepProps) => {
         innerRef={formikRef}
         enableReinitialize={true}
         onSubmit={(values: IProject) => {
-          return onSubmitReview(values, formikRef, submitStatusCode).then((project: IProject) => {
-            if (project?.statusCode === ReviewWorkflowStatus.ApprovedForErp) {
-              goToDisposePath('../approved');
-            }
-            if (project?.statusCode === ReviewWorkflowStatus.Denied) {
-              goToDisposePath('../summary');
-            }
-          });
+          const workflowCode =
```

Comment:
use the workflow code of the project by default. TODO: figure out if we need to get the NEXT workflow if the next status is not in this workflow.